gooddata-cn-on-eks/variables.tf

```diff
@@ -139,4 +139,16 @@ variable "tags" {
   description = "Tags to be added to resources"
   type        = map(string)
   default     = {}
-}
\ No newline at end of file
+}
+
+variable "enable_kube_prometheus_stack" {
```

Comment:
Please make the `helm_release.kps_crds` and also `pulsar_dashboard` and `gooddata_dashboard` ConfigMaps installed only when this variable is set to true.